.circleci/config.yml

```diff
@@ -351,8 +316,6 @@ jobs:
       - *attach_release_output
       - *setup_bazel_ci_config
       - *setup_bazel_remote_execution
-      - *yarn_download
-      - *yarn_install
       - *setup_bazel_binary
```

Comment:
we actually need the yarn download and yarn install here as we need to download bazel through yarn. 

Afterwards it will be linked globally using the `setup_bazel_binary`.

Reply:
haha no worries 😄 can you elaborate why we want to remove `yarn_download` though?

The reason we have this is that we can control the yarn version better without having to depend on a specific version coupled with the Docker image. We did the same on `angular/angular` for that reason.

Reply:
Yeah, I'd vote for keeping it like that to still have the flexibility. Up to you though. I don't feel too strong about it, but IIRC we already had a discussion about this and decided to download Yarn separately.